libpod/container_internal_linux.go

```diff
@@ -118,7 +124,13 @@ func (c *Container) prepare() error {
 	wg.Wait()
 
 	var createErr error
+	if err := c.setupOverlayMounts(); err != nil {
```

Comment:
This is already part of mountContainer, why do we need it here?

Reply:
This is `preparing` the directories required for the overlay mount.  Where do you think it should go?